samples/snippets/src/test/java/com/example/asset/ListAssets.java

```diff
@@ -64,4 +64,18 @@ public void testListAssetsExample() throws Exception {
       assertThat(got).contains("asset");
     }
   }
+
+  @Test
+  public void testListAssetsRelationshipExample() throws Exception {
```

Comment:
Replace with more precise Exception sub-type.

Reply:
Add in the last commit